scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -16,20 +16,35 @@ This notebook will demonstrate how to:
-This project performed single-cell sequencing of pediatric solid tumors, including the rhabdomyosarcoma (RMS) samples we'll be looking at here ([Patel _et al._ (2022)](https://doi.org/10.1016/j.devcel.2022.04.003)).
+Specifically, we'll have a look at four libraries from the [`SCPCP000005` project](https://scpca.alexslemonade.org/projects/SCPCP000005; [Patel _et al._ (2022)](https://doi.org/10.1016/j.devcel.2022.04.003)), an investigation of pediatric solid tumors led by the [Dyer](https://www.stjude.org/research/labs/dyer-lab.html) and [Chen](https://www.stjude.org/research/labs/chen-lab-taosheng.html) labs at St. Jude Children's Research Hospital.
+The particular libraries we'll integrate come from two rhabdomyosarcoma (RMS) patients, with two libraries (each from separate samples) from two patients each.
+All library are single-nuclei and were sequenced with 10Xv3 technology.
 
+We'll be integrating these libraries with two different tools, [`fastmNN`](http://www.bioconductor.org/packages/release/bioc/html/batchelor.html; [Haghverdi _et al._ (2018)](https://doi.org/10.1038/nbt.4091)) and [`harmony`](https://portals.broadinstitute.org/harmony/; [Korsunsky _et al._ (2019)](https://doi.org/10.1038/s41592-019-0619-0)). 
+Integration corrects for batch effects that arise from different library preparations, genetic backgrounds, and other sample-specific factors, so that datasets can be jointly analyzed. 
+`fastMNN` corrects for batch effects using a faster variant of the mutual-nearest neighbors algorithm, the technical details of which you can learn more about from this [vignette by Lun (2019)](https://marionilab.github.io/FurtherMNN2018/theory/description.html).
+`harmony`, on the other hand, corrects for batch effects using an iterative clustering approach, and unlike `fastMNN`, it is also able to consider additional covariates beyond just the batch groupings.
 
 
-We'll be integrating these libraries with two different tools, [`fastmNN`](http://www.bioconductor.org/packages/release/bioc/html/batchelor.html) and [`harmony`](https://portals.broadinstitute.org/harmony/). 
-**Still fleshing this out**
+Regardless of which integration tool is used, the SCE objects to be integrated need to be reformatted and merged into a single (uncorrected!) SCE object that contains all cells from all SCES for all genes that the SCEs have in common.
+This merged SCE, which should also contain explicit information about batch origins for each cell (here, the originating library), can be provided to the integration method to obtain a formally batch-corrected SCE object. 
+
+Before we dive into this, let's first build a sense of what this merging process looks like.
```

Comment:
I actually kind of think we want to talk and show this right before we do it? So this should be the image that's at the beginning of the merging section? Alternatively (or maybe both?), the image you made should be in slides and I think we would have just talked about it in slides right before this so I would start by setting up the notebook, introducing the functions and tools you need to introduce, and then bring back the image right before you merge.

Reply:
I agree that it makes sense for the image I made to be in slides. Maybe I put the annotated screen shot in slides _and_ in the Rmd, for both overall introduction and reference during coding?

That said, it's looks to me like that text is too small to fully see _within_ the Rmd during development, which is how participants will realistically see these images (in other words, they're not looking at the rendered HTML here for the most part). 
We can handle that sizing issue with that either custom CSS or using `[indicate the css class in here to set image size](path/img.png)` syntax, or with a `knitr::include_graphics()` chunk, if anyone has a preference?